### PR TITLE
Day 1 updating 1.0 after install

### PR DESCRIPTION
**Pull request by ndufreche**
Day 1 - 1.0 update

### CODE REVIEW

**Comment by eric-chau** [comment]
It seems good, I guess you already tried to install it and it works! So you have my :+1:

**Comment by ndufreche** [comment]
Yes is tested :)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+composer update
+
+cd public
+
+php -S localhost:8080
+```
+
+Into your favorite navigator go to this page `localhost:8080/install.php`
+
+Complete each form and follow instruction.
```
Comment (mickaelandrieu) [inline README.md]: the instructions.

**Comment by mickaelandrieu** [comment]
thank you !

**Pull request merged**